src/ert/storage/local_experiment.py

```diff
@@ -269,13 +269,17 @@ def parameter_configuration(self) -> Dict[str, ParameterConfig]:
             params[data["name"]] = _KNOWN_PARAMETER_TYPES[param_type](**data)
         return params
 
-    @cached_property
+    @property
     def response_configuration(self) -> Dict[str, ResponseConfig]:
-        params = {}
+        responses = {}
         for data in self.response_info.values():
-            param_type = data.pop("_ert_kind")
-            params[data["name"]] = _KNOWN_RESPONSE_TYPES[param_type](**data)
-        return params
+            ert_kind = data.pop("_ert_kind")
```

Comment:
Maybe drop `_ert_kind` and just use `type`?

Reply:
Hmm could, by `type` you mean `response_type`? In the case of a "dynamic" `response_type` with `CSV_RESPONSE` / `CSV_OBSERVATION` we would need some way to identify the class, and `_ert_kind` is the already existing way to do this. If we know for sure we can lock in response type (which is currently used to identify names of tables of responses in storage) we could go for it. Maybe leave this for a separate PR?